l1-contracts/deploy-scripts/InteropLibrary.sol

```diff
@@ -248,27 +259,40 @@ library InteropLibrary {
             revert ZeroAddress();
         }
 
-        InteropCallStarter[] memory calls = new InteropCallStarter[](1);
-        calls[0] = buildSendCall({
+        InteropCallStarter[] memory calls = new InteropCallStarter[](1); // merge then call and bundle attrs
+        bytes[] memory bundleAttributes;
+        (calls[0], bundleAttributes) = buildCall({
             destinationChainId: destination,
             target: target,
             executionAddress: executionAddress,
             unbundlerAddress: unbundlerAddress,
             data: data
         });
 
+        bytes[] memory mergedAttributes = new bytes[](calls[0].callAttributes.length + bundleAttributes.length);
```

Comment:
can you put this merging in a separate function please so that it does not clutter things up